<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>871</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QWidget" name="principal">
    <widget class="QPushButton" name="SELECCION_FILAS_COLUMNAS">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SALTAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PASO_AVANCE">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>360</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>AVANZAR</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Seleccion">
    <widget class="QTableWidget" name="Bloques">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>351</width>
       <height>361</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>169</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>NOMBRE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ESTADO</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="Avanzar">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Avanzar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Retroceder">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retroceder</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="Pasos">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>461</width>
       <height>361</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>148</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>NOMBRE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ESTADO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>COMANDO</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
